profile/README.md

```diff
@@ -0,0 +1,53 @@
+
+#### Benefits of Open Data Hub 
+
+ODH is a meta-project that integrates open source projects into a practical solution. It aims to foster collaboration between communities, vendors, user-enterprises, and academics following open source best practices. The open source community can experiment and develop intelligent applications without incurring high costs and having to master the complexity of modern machine learning and artificial intelligence software stacks. 
+
+**Contribution Guidelines**:  
+[https://github.com/opendatahub-io/opendatahub-community/blob/master/contributing.md](https://github.com/opendatahub-io/opendatahub-community/blob/master/contributing.md)
+
+**Release notes**:  
+[https://opendatahub.io/docs/roadmap/release-notes.html](https://opendatahub.io/docs/roadmap/release-notes.html)
+
+**Roadmap**:  
+[https://opendatahub.io/docs/roadmap/future.html](https://opendatahub.io/docs/roadmap/future.html)
+
+**Getting Started**:  
+[https://opendatahub.io/docs/getting-started/quick-installation.html](https://opendatahub.io/docs/getting-started/quick-installation.html)
+
+##
+
+### Repositories
+
+ - **Core/Main Repositories**
+	 * [odh-manifests](https://github.com/opendatahub-io/odh-manifests): A repository for Open Data Hub components Kustomize manifests. 
+	 * [opendatahub-operator](https://github.com/opendatahub-io/opendatahub-operator): A repository that helps to deploy, monitor and manage the lifecycle of Kubeflow. Built using the Operator Framework which offers an open source toolkit to build, test, package operators and manage the lifecycle of operators.
+	 * [odh-dashboard](https://github.com/opendatahub-io/odh-dashboard): A dashboard for Open Data Hub components. Shows what's installed and what's available for installation, as well as, links to component UIs and links to component documentation.
+	 * [kfp-tekton](https://github.com/opendatahub-io/kfp-tekton): Project bringing Kubeflow Pipelines and Tekton together. The current code allows you run Kubeflow Pipelines with Tekton backend end to end.
+
+  - **Tools**
+  	 *  [kubeflow](https://github.com/opendatahub-io/kubeflow): This repository contains the code of kubeflow the cloud-native platform for machine learning operations - pipelines, training and deployment.
+  	 *  [manifests](https://github.com/opendatahub-io/manifests): The Kubeflow Manifests repository is organized under three (3) main directories, which include manifests for installing: `apps`,  `common`, `contrib`
+  	 *   [odh-images](https://github.com/opendatahub-io/odh-images):  This repository contains code for the various Open Data Hub images.
```

Comment:
I would remove this line since it will probably be moved along with the other Tier 2 repos.